src/components/search/search.js

```diff
@@ -78,13 +86,29 @@ class Search extends React.Component {
 }
 
 Search.propTypes = {
-  placeholder: PropTypes.string, // option to replace the input placehoder with a different string,
-  narrow: PropTypes.bool, // option to collapse input to fit in a crowded space
+  /** replace the input placehoder with a different string */
+  placeholder: PropTypes.string,
+  /** collapse input to fit in a crowded space */
+  narrow: PropTypes.bool, //
   background: PropTypes.oneOf(['light', 'dark']),
-  inputId: PropTypes.string, // option to override default id for input/label, used for testing
-  disableModal: PropTypes.bool, // option to completely disable modal if you always want an input
-  site: PropTypes.string, // option to add current site or all docs filter toggle
-  connector: PropTypes.instanceOf(SiteSearchAPIConnector) // option to connect to a custom search engine
+  /** override default id for input/label, used for testing */
+  inputId: PropTypes.string,
+  /** disable modal if you always want an input */
+  disableModal: PropTypes.bool,
+  /** add current site filter toggle */
+  site: PropTypes.string,
+  /** option to connect to a custom search engine */
+  connector: PropTypes.instanceOf(SiteSearchAPIConnector),
+  /** If true, only show results from search */
+  resultsOnly: PropTypes.bool,
+  /** If `resultsOnly: true` set `overrideSearchTerm` to set the search term */
+  overrideSearchTerm: PropTypes.string,
+  /** Segment track event, default is (Searched docs) */
+  segmentTrackEvent: PropTypes.string,
+  /** If true, enable compact mode utilizing smaller text and padding, default false */
+  themeCompact: PropTypes.bool,
+  /** Node to display when there are no search results for the given query */
+  customEmptyResultMessage: PropTypes.node
```

Comment:
Awesome!

Reply:
@adrianababakanian I think we could rename this prop `emptyResultMessage` and then set its defaultProps in search.js like so:

```jsx
Search.defaultProps = {
emptyResultMessage: (
    <p>
      Hmmm, we didn't find anything. Reword your
      search, or{' '}
      <a href="https://support.mapbox.com/hc/en-us">
        contact Support
      </a>
      .
    </p>
  )
}
```

Then in search-box.js, we can reference `{props.emptyResultMessage}` since the default will be set in search.js.

Reply:
Great suggestion @katydecorah , I like that pattern better! I'll incorporate this change.